software/digistorm.yml

```
@@ -0,0 +1,11 @@
+name: "Digistorm"
+website_url: "https://ladigitale.dev/"
```

```suggestion
website_url: "https://digistorm.app/"
```

Let's use the main instance as `website_url`, since it is available in English and correctly describes what the project does.

---

sounds good, will do that for future ones!